spec/models/iiif_canvas_indexer_spec.rb

```diff
@@ -19,7 +19,7 @@
       id: druid,
       content_metadata_type_ssm: ['image'],
       iiif_manifest_url_ssi: 'http://example.com',
-      identifier_ssim: %w(ABC 123 xyz)
+      manuscript_number_tesim: %w(ABC 123 xyz)
```

Comment:
@jkeck should this be changed to a single string, like e.g. `MS 079`? The test still works, but it might make it easier to scan this spec for expected behavior.

Reply:
👌 It still has to be an array because it's a multi-valued field in solr, but I've added a more realistic value into that array.